src/FeatureflowProvider.js

```diff
@@ -1,33 +1,45 @@
 import React, { Component } from 'react';
-import { featureflowClientShape, featureflowConfigShape } from './PropTypes';
-
+import {Provider} from "./context";
 
 const defaultFeatureflowConfig = {
   update: false,
   clientName: 'featureflow'
 };
 
-
 export default class FeatureflowProvider extends Component{
```

Comment:
could make this a nice function component instead and use `useEffect` instead of `componentWillMount` etc

Reply:
👍 
will do exactly that